strings/tips.txt

```diff
@@ -248,3 +248,4 @@ When hacking doors, cutting and mending a "test light wire" will restore power t
 When crafting most items, you can either manually combine parts or use the crafting menu.
 Suit storage units not only remove blood and dirt from clothing, but also radiation!
 Remote devices will work when used through cameras. For example: Bluespace RPEDs and door remotes.
+You can light a cigar on a supermatter crystal.
```

Comment:
I feel like making this more vague would be more helpful

`You can light a cigar on anything if you're brave enough`

Reply:
I like that tip, though it's decidedly not true and I think the matter-of-factness of this one has a comedic value of its own.